### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds a table with a detailed explanation of the intent of each category.

### CODE REVIEW

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks @schalkneethling for this important addition! 👍 

Is it in any way possible to have this table right on the [discussions](https://github.com/mdn/mdn-community/discussions) page? (also this readme is getting too long with a lot of important messages). It might be more efficient to have the table on the discussions page to avoid toggling with this page to see the definitions.
Better still, can each category on the discussions page have a statement describing the intent of the category - that will make the contributors stop and think if they're in the right place even if they've missed to check this table of definitions.

**Comment by schalkneethling** [comment]
> It might be more efficient to have the table on the discussions page to avoid toggling with this page to see the definitions.

That is unfortunately not possible at the moment.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ At MDN, we put people first and do our best to recognize, appreciate, and respec
...
+| 👾  [MDN Plus](https://github.com/mdn/mdn-community/discussions/categories/mdn-plus) | Discussions related to the [existing MDN Plus feature set](https://developer.mozilla.org/en-US/plus) as well as feature ideas. For MDN Plus support, please refer to our [official support channel](https://support.mozilla.org/en-US/products/mdn-plus). |
+| 🛠️  [Platform](https://github.com/mdn/mdn-community/discussions/categories/platform) | Discussions related to the [core](https://github.com/mdn/yari) MDN Web Docs platform. These discussions are meant as a means of discovery and exploration of possible ways to improve the architecture and existing features. _NOTE:_ This is not for MDN Plus related feature discussions. If you believe you found a bug, please [report it](https://github.com/mdn/yari/issues/choose) on the Yari repository. |
```
[the remark this answers is not in the document]
Comment (schalkneethling) [reply README.md:30]: > I guess 'discovery and exploration' part is applicable to all discussions?

Sure but here I wanted to call that out to make it clear it is not for bug reports.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,28 @@ At MDN, we put people first and do our best to recognize, appreciate, and respec
 
 > **Note:** By participating in and contributing to our projects and discussions, you acknowledge that you have read and agree to the [Mozilla community participation guidelines](https://github.com/mdn/mdn-community/blob/main/CODE_OF_CONDUCT.md).
 
+On MDN Web Docs, we encourage our community to start and/or engage in discussions around topics related to the overall project. Discussions are categorized by different topics areas. We ask that you keep each discussion focused on the topic at hand, instead of covering multiple topics in one discussion.
+
+_NOTE:_ mdn-community/discussions is not the place to report problems. For content-related probelms you come across on MDN Web Docs, it is best to raise issues against the [relevant project](https://github.com/mdn/). If you're ever in doubt about whether to open an issue or a discussion, consider the following guidelines:
+
+- Issues are for reporting a bug with a clearly defined and actionable outcome.
+- If the issue needs a discussion to agree upon a course of action or define an actionable piece of work, you are in the right place.
```
Comment (schalkneethling) [inline README.md:16]: How about:

```suggestion
- If the issue needs a discussion to agree upon a course of action or to define an actionable piece of work, you are in the right place to initiate a discussion.
```

File: README.md [hunk 3]
```diff
@@ -8,6 +8,28 @@ At MDN, we put people first and do our best to recognize, appreciate, and respec
...
+
+Check out the definition of each discussion category below so that you can start your discussion in the proper place.
+
+| **Feedback Category** | **Category subject** 	|
```
Comment (schalkneethling) [inline README.md:20]:
```suggestion
| **Discussion category** | **Category subject** 	|
```

File: README.md [hunk 4]
```diff
@@ -8,6 +8,26 @@ At MDN, we put people first and do our best to recognize, appreciate, and respec
...
+|---	|---	|
+| 📣  [Announcements](https://github.com/mdn/mdn-community/discussions/categories/announcements) | This category is reserved for MDN Web Docs staff. While there is nothing in place to prevent others from posting here, we ask that you choose one of the other available categories. |
+| 🔮  [Browser compatibility data](https://github.com/mdn/mdn-community/discussions/categories/browser-compatibility-data) | Discussions related to the [browser-compat-data](https://github.com/mdn/browser-compat-data) project. |
+| ✏️  [Content](https://github.com/mdn/mdn-community/discussions/categories/content) | Discussions related to the [content](https://github.com/mdn/content) on MDN Web Docs. However, this is not the place to ask for help with code examples. For this, please [join our community on Discourse](https://discourse.mozilla.org/c/mdn/learn/250). |
+| 🙋  [Content suggestions](https://github.com/mdn/mdn-community/discussions/categories/content-suggestions) | If you have an idea for new or improvements to existing content on MDN Web Docs, this is the place to start your discussion. We commonly label these as opportunity assessments. |
```
[the remark this answers is not in the document]
Comment (schalkneethling) [reply README.md:25]: Yeah, we will label some of the discussions. This is one of those we might label as such.

**Comment by schalkneethling** [comment]
> can each category on the discussions page have a statement describing the intent of the category

You can have short blurbs, but nothing as long as we have here. I believe the intent is to have a place to point people to should we need to clarify. If people end up reading it beforehand, that is a bonus. Once this is merged, we can post an announcement and pin it to the discussions page.

Something like, "Please read this before starting a discussion" as the subject line.

File: README.md (hunk 2 above)
Comment (dipikabh) [reply README.md:16]: Looks good

**Review by dipikabh — APPROVED** [review]

**Pull request merged**